plugins/modules/monitoring/datadog/datadog_monitor.py

```diff
@@ -111,9 +111,16 @@
     thresholds:
         description:
           - A dictionary of thresholds by status.
-          - Only available for service checks and metric alerts.
+          - Only available for service checks, metric alerts, and query alerts.
           - Because each of them can have multiple thresholds, we do not define them directly in the query.
         default: {'ok': 1, 'critical': 1, 'warning': 1}
+    threshold_windows:
+        description:
+          - A dictionary of threshold windows by status
+          - These options only apply to anomaly monitors which are of the "query alert" type; Use elsewhere will be ignored.
+          - "recovery_window: Describes how long an anomalous metric must be normal before the alert recovers"
+          - "trigger_window: Describes how long a metric must be anomalous before an alert triggers"
```

Comment:
```suggestion
          - "recovery_window: Describes how long an anomalous metric must be normal before the alert recovers"
          - "trigger_window: Describes how long a metric must be anomalous before an alert triggers"
```
what types of values are acceptable?

Reply:
@Tyler-2 ^ , and what about adding an example (at least one)?

Reply:
I believe I have addressed both of these comments. Please resolve conversation if you agree.